app/controllers/health_card_controller.rb

```diff
@@ -0,0 +1,26 @@
+require 'health_cards'
+
+class HealthCardController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:create]
+
+  def index; end
+
+  def create
+    fhir_bundle = FHIR.from_contents(session[:fhir_bundle])
+    vaccine_group = params[:vaccine_group] || 'COVID-19'
+
+    health_card = HealthCard.new(fhir_bundle)
```

Comment:
I think you also need to add an `issuer:` parameter to the constructor here. Right now the `iss:` field of the SHC is `null`, and it needs to be a string.

Reply:
This isn't a `HealthCards::HealthCard`, it's a different class [(see here)](https://github.com/dvci/iis-shc-issuer/blob/ef4d8f5f508517e1a7fced8e965dfc7d46c63364/app/models/health_card.rb).

This class name should change to avoid confusion since this class is really about representing the data returned from the API more than representing a HealthCard.

Reply:
When I download the QR code and scan it with the Microsoft Verifier Portal, I get the following error:
`· Schema: /iss error : must be string (Error)`, since the field is displayed as `null`, so I'm assuming somewhere we need to pass in the issuer URL to `HealthCards::HealthCard`. It looks like this is done in [config/initializers/smart_health_cards.rb](https://github.com/dvci/iis-shc-issuer/blob/dvci-147-smart-health-card/config/initializers/smart_health_cards.rb#L9), but i can't find where the `HOST` environment variable is actually set in our environment. Should we add this to `.env.development`? I just tried this locally and it properly adds the `iss:` field when I verify the newly generated QR code.